spec/features/guide_publishing_flow_spec.rb

```diff
@@ -61,12 +63,56 @@
     end
   end
 
+  context "with a review requested" do
+    it "lists editions that need a review" do
+      edition = Generators.valid_edition(approvals: [])
+      guide = Guide.create!(latest_edition: edition, slug: "/service-manual/something")
+
+      visit edit_guide_path(guide)
+      click_button "Send for review"
+      visit guides_path
+      expect(page).to have_content "review_requested"
+    end
+
+    context "approved by another user" do
+      it "lists editions that are approved" do
+        edition = Generators.valid_edition(state: "review_requested", approvals: [])
+        guide = Guide.create!(latest_edition: edition, slug: "/service-manual/something")
+
+        reviewer = User.new(name: "Some User")
+        login_as reviewer
+        visit guides_path
+        click_link "Create new edition"
+        click_button "Mark as Approved"
+        expect(page).to have_content "Thanks for approving this guide"
+        expect(page).to have_content "approved"
+      end
+    end
+  end
+
+  context "without any approvals" do
+    it "does not allow guides to be published" do
+      edition = Generators.valid_edition(state: "review_requested", approvals: [])
+      guide = Guide.create!(latest_edition: edition, slug: "/service-manual/something")
+      visit edit_guide_path(guide)
+      expect(page).to_not have_button "Publish"
+    end
+  end
+
```

Comment:
I think we should have a test to prove the original author can not approve their own changes